gateway/admin.go

```diff
@@ -19,8 +19,8 @@ var prefix = "gw_jobs_"
 func (g *GatewayAdmin) Status() interface{} {
 	configSubscriberLock.RLock()
 	defer configSubscriberLock.RUnlock()
-	writeKeys := make([]string, 0, len(enabledWriteKeysSourceMap))
-	for k := range enabledWriteKeysSourceMap {
+	writeKeys := make([]string, 0, len(writeKeysSourceMap))
+	for k := range writeKeysSourceMap {
 		writeKeys = append(writeKeys, k)
```

Comment:
Since the field's name is `enabled-write-keys`, we should only include enabled write keys.
If we are interested in including in the status response disabled key too, then I suppose we should include an additional `disabled-write-keys` field.

Reply:
but, that is the reason why I have renamed it to `writeKeysSourceMap` from `enableWriteKeysSourceMap`. Since, now it will have all the write keys (enabled & disabled both).